lib/data/external/gcp-2017-11-01.api.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Could we put all these base files at are loose in the `external` folder into their own `GCP` folder? Like `external/GCP`. I think we should try to keep things as easy to navigate/manage as possible :)

Reply:
I see. Thanks for the tip. I will move the files into their own folder like you mentioned.